src/simuq/ionq/ionq_api_circuit.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+from simuq.backends.ionq_circuit import IonQCircuit, to_turns
+
+
+class IonQAPICircuit(IonQCircuit):
+    def __init__(self, n_qubits, name, backend="simulator", noise_model=None):
+        super().__init__(n_qubits)
+        # Circuit are stored in turns, accum_phases are stored in rads.
+        self.job = {
+            "lang": "json",
+            "shots": 4096,
```

Comment:
Seems like a dangerously high shot count. This is ultimately overwritten later on but I'd make this smaller for safety

Reply:
In retrospect, this probably also means the default shot count in the IonQProvider should be adjusted downward

Reply:
Maybe shots = 0 here and 10 in the IonQProvider? @PicksPeng for visibility